### DISCUSSION TITLE
Anyone interested in building @graphiql/vue?

### DISCUSSION THREAD

**Discussion by Sytten** [graphql/graphiql#3178] [General]
React is currently the only implementation, but other implementations would be useful to the community to avoid loading a full react application in case you want to embed it an existing vue application.

Wondering if someone had started work on that or would be interested in participating in the effort!

**Comment by acao** [comment]
@Sytten great question! it has been asked about in discord several times, and in some closed issues, so you could find some others interested I'm sure!

I would reccomend starting with `monaco-graphql` instead of `codemirror-graphql`. 

The `monaco-graphl-webpack` example I built in vanillajs for exactly this reason, to show how to implement it without react, and the `monaco-react-vite example` could be ported to vue readily for a simple starting point!

it might seem tempting to just port `@graphiql/react` more closely, but that library is about to be almost entirely rewritten for monaco editor & 3.0.0!
